### PR TITLE
Ansible changes

### PR DESCRIPTION
**Pull request by nirarg**

### CODE REVIEW

File: ansible/playbook.yml [hunk 1]
```diff
@@ -34,25 +34,66 @@
...
+  - name: Check if cluster exists
+    stat:
+      path: "{{ansible_user_dir}}/ansible"
+    register: ocp_cluster
 
-  - name: Save pull secret file
-    ansible.builtin.copy:
-      src: "{{ pull_secret_file }}"
-      dest: "{{ pull_secret_remote_path }}"
+  - name: Clean OCP cluster if exists
```
Comment (ygalblum) [inline ansible/playbook.yml]: You can merge these two tasks into one block (https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_blocks.html). This way you can check `ocp_cluster.stat.exists` only once

File: environment [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```
Comment (ygalblum) [inline environment:1]: Since you already use the `FOO="${FOO:-default value}"` notation, you can lave the setting of these fields inside the script

File: README.md [hunk 3]
```diff
@@ -41,6 +41,7 @@ You can either use the provided `Ansible` playbook or run the steps manually
           ansible_user: <Username to connect with>
           ansible_ssh_private_key_file: <Location of your Private SSH key>
           ansible_become: <Set to false if connecting as "root", otherwise remove>
+          ansible_python_interpreter: /usr/libexec/platform-python
```
Comment (ygalblum) [inline README.md:44]: Why did you need to set this?

File: ansible/playbook.yml (hunk 1 above)
Comment (nirarg) [reply ansible/playbook.yml]: Nice, thanks for the tip .... changed it

File: README.md (hunk 3 above)
Comment (nirarg) [reply README.md:44]: This is done in order to use the same python version, even if the python command in PATH changed to another version (modified by dev-scripts)
